From 2ac8183e48a0ab95fa3f405bc26b1b20d51755f2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?V=C3=A1clav=20Slav=C3=ADk?= <vslavik@gmail.com>
Date: Sun, 27 Oct 2013 09:17:45 +0100
Subject: [PATCH] Fix accidental use of operator+ instead of operator<<.

There was a typo in the output expression, appending a number to a string, instead of concatenating them as indented.
---
 src/core/index/CheckIndex.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/index/CheckIndex.cpp b/src/core/index/CheckIndex.cpp
index 9e910f1..7f783d8 100644
--- a/src/core/index/CheckIndex.cpp
+++ b/src/core/index/CheckIndex.cpp
@@ -640,7 +640,7 @@ namespace Lucene
                 std::wcout << L"WARNING: would write new segments file, and " << result->totLoseDocCount << L" documents would be lost, if -fix were specified\n\n";
             else
             {
-                std::wcout << L"WARNING: " << result->totLoseDocCount + L" documents will be lost\n";
+                std::wcout << L"WARNING: " << result->totLoseDocCount << L" documents will be lost\n";
                 std::wcout << L"NOTE: will write new segments file in 5 seconds; this will remove " << result->totLoseDocCount;
                 std::wcout << L" docs from the index. THIS IS YOUR LAST CHANCE TO CTRL+C!\n";
                 for (int32_t sec = 0; sec < 5; ++sec)
-- 
1.8.5.2

